ChangeSet
  1.1548 04/04/01 14:50:45 dgibson@wocket.ozlabs.ibm.com +1 -0
  This patch adds a debugging message to the ppc64 hugepage code when we
  attempt to open the "low" (32-bit) hugepage window on PPC64, but can't
  because a (non-hugepage) mapping already exists in the region.

  arch/ppc64/mm/hugetlbpage.c
    1.13 04/04/01 14:50:18 dgibson@wocket.ozlabs.ibm.com +4 -1
    This patch adds a debugging message to the ppc64 hugepage code when we
    attempt to open the "low" (32-bit) hugepage window on PPC64, but can't
    because a (non-hugepage) mapping already exists in the region.

diff -Nru a/arch/ppc64/mm/hugetlbpage.c b/arch/ppc64/mm/hugetlbpage.c
--- a/arch/ppc64/mm/hugetlbpage.c	Thu Apr  1 09:18:51 2004
+++ b/arch/ppc64/mm/hugetlbpage.c	Thu Apr  1 09:18:51 2004
@@ -253,8 +253,11 @@
 	/* Check no VMAs are in the region */
 	vma = find_vma(mm, TASK_HPAGE_BASE_32);
 
-	if (vma && (vma->vm_start < TASK_HPAGE_END_32))
+	if (vma && (vma->vm_start < TASK_HPAGE_END_32)) {
+		printk(KERN_DEBUG "Low HTLB region busy: PID=%d  vma @ %lx-%lx\n",
+		       current->pid, vma->vm_start, vma->vm_end);
 		return -EBUSY;
+	}
 
 	/* Clean up any leftover PTE pages in the region */
 	spin_lock(&mm->page_table_lock);
.........................................................................
# vim: syntax=diff

